module/VuFind/src/VuFind/AjaxHandler/GetItemStatuses.php

```diff
@@ -366,18 +373,17 @@ protected function getItemStatusGroup($record, $messages, $callnumberSetting)
     {
         // Summarize call number, location and availability info across all items:
         $locations = [];
-        $use_unknown_status = $available = false;
+        $useUnknownStatus = false;
+        $available = 'false';
         foreach ($record as $info) {
             // Find an available copy
-            if ($info['availability']) {
-                $available = $locations[$info['location']]['available'] = true;
+            if ($info['availability'] && 'true' !== $available) {
```

Comment:
Might this check cause a short-circuit that prevents $locations[$info['location']] from getting populated? Looks to me like if the first location is available, then we won't fill in the details for subsequent locations, but maybe I'm missing something.

Reply:
Oops, fixed.